src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String ADDED = "buy";
+    private static final String LOSE = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method looks big, let's split it into smaller ones (for example writing to file, reading from file, forming result)